django-stubs/db/models/functions/datetime.pyi

```diff
@@ -41,14 +41,14 @@ class TruncBase(TimezoneMixin, Transform):
     tzinfo: Any
 
     def __init__(
-        self, expression: Expression, output_field: Field | None = ..., tzinfo: tzinfo | None = ..., **extra: Any
+        self, expression: Combinable | str, output_field: Field | None = ..., tzinfo: tzinfo | None = ..., **extra: Any
```

Comment:
I'm not very familiar QuerySet operators/functions.

Most `Combinable`s seem to inherit from `Expression`. Can you give an example of something that is `Combinable` but not `Expression`?

Reply:
Also some PostgreSQL functions are typehinted as `BaseExpression | Combinable | str`. I wonder if there's a reason for that, or in practice all useful expressions already inherit from `Combinable`.

Reply:
Sry, you already answered the first question in PR description. `F()` objects inherit from `Combinable` but not `Expression`.

Reply:
> I wonder if there's a reason for that, or in practice all useful expressions already inherit from `Combinable`.

But unless someone can think of a use case for `BaseExpression`, I'm inclined to go with `Combinable | str`. Let's not complicate things unless we know it's necessary.